java-auto-instrumetation/cmd/manager/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
```

Comment:
In principle, there should be 3 sections in the import:
- system libraries
- external packages
- internal packages

Reply:
👍